Methods Paper Writing/01-SYTO9-PI-Concentration-Optimization/Analysis/SYTO9-PI-Concentration-Graphing.rmd

```diff
@@ -33,37 +40,53 @@ oriNames <- sampleNames(fs)
 tmp <- str_split(oriNames, pattern = "[ _]+", simplify = TRUE)[,c(1, 5, 6, 7)] 
 colnames(tmp) <- c("Date", "Treatment", "Dilution", "Dye") 
 sample <- data.frame(tmp) %>% 
-  mutate(`Dye` = ifelse(`Dye` == "p.fcs", "PI", ifelse(`Dye` == "", "SYTO9", `Dye`)), Dilution = gsub(".fcs", "", Dilution))
+  mutate(Dye = ifelse(Dye == "p.fcs", "PI", "SYTO9"), 
+         Dilution = gsub(".fcs", "", Dilution) %>% as.integer(),
+         Treatment = factor(Treatment, levels = as.character(c(0, 10, 100, 1000))))
 rownames(sample) <- oriNames
 pData(fs) <- sample
 print(pData(fs))
 ```
-```
 
 # Visualize dataset
+
 ```{r}
 sub <- fs[pData(fs)$Dye == "PI" & pData(fs)$Date == "112023" & pData(fs)$Treatment == "0"]
 p1 <- ggcyto(sub, aes(x = "BL1.H", y = "BL3.H")) + geom_hex(bins = 80) + 
   facet_wrap(~Dilution, labeller = "label_both") + theme_minimal()
 p1 + scale_x_logicle() + scale_y_logicle()
 ```
 
+```{r}
+sub <- fs[pData(fs)$Dye == "SYTO9" & pData(fs)$Date == "112023"]
+p2 <- sub %>% 
+  ggcyto(aes(x = "BL1.H", y = "BL3.H")) + 
+  geom_hex(bins = 80) + 
+  facet_grid(Dilution ~ Treatment, labeller = "label_both") + 
+  theme_minimal()
+p2 + scale_x_logicle() + scale_y_logicle()
+```
 
 # Extract MFI for BL1.H across all flowFrames
 ```{r}
-```{r}
-library(flowCore)
 tmp <- fsApply(fs, each_col, median)
-combined_data <- cbind(pData(fs), tmp)
-pData(fs) <- combined_data
-print(pData(fs))
+if(all(rownames(tmp) == rownames(sample))){
+  combined_data <- cbind(pData(fs), tmp) %>% 
+    as.tibble() %>% relocate(name, .after = last_col())
+}
+# I don't recommend assigning the combined_data to the phenoData slot of the
+# fs object. this is not necessary and makes the phenoData slot complicated
+# instead, just work with the combined_data object you created
+#pData(fs) <- combined_data
+#print(pData(fs))
+print(combined_data)
 ```
 
 # Graph Green MFIs against teatment and graoup by dye concentration
```

Comment:
I didn't change the plotting code below because I need to know what point you want to make with this plot.